### DISCUSSION TITLE
요론 게 생겼네

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#7] [General]
![캡처](https://user-images.githubusercontent.com/73771162/190887800-0fd0adf5-d49c-432c-a1c8-03270f50862e.PNG)

백로그에 생긴 이슈
자동으로 생긴 거?
직접 연습하신 거?

**Comment by Lcat12345** [comment]
푸하하! 직접 생성한거였습니다~

> 예에
